### PR TITLE
Release 9.2

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
Upgraded version number and some other fixes. Please notice that the changes in the package construction DO NOT AFFECT the released apps. Only the ones that are compiled internally.

### CODE REVIEW

File: scripts/wheels/windows/build.ps1 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 param([System.String]$cotire="OFF")
 
 $pythons = "36", "37", "38", "39"
```
Comment (loumalouomega) [inline scripts/wheels/windows/build.ps1:3]: With the last pybind11 should be 3.7 minimum

File: scripts/wheels/setup.py [hunk 2]
```diff
@@ -39,6 +39,31 @@ def replaceKeywords(stringArray):
     f.writelines(list(map(lambda x: replaceKeyword(x) + "\n", conf["excluded_binaries"])))
     f.close()
 
+package_classifiers = [
+        "Programming Language :: Python :: 3",
+        "Programming Language :: C++",
+        "Programming Language :: Python",
+        "Programming Language :: Python :: 3.6",
```
Comment (loumalouomega) [inline scripts/wheels/setup.py:46]: Idem pybind11 is stating with 3.7

Comment (roigcarlo) [reply scripts/wheels/setup.py:46]: Ok, I will update for all packages after merging this :+1:

Comment (roigcarlo) [reply scripts/wheels/setup.py:46]: But we are still able to generate 3.6 packages correctly, what is the problem?

Comment (loumalouomega) [reply scripts/wheels/setup.py:46]: If it works, it works. But there is no official support anymore

**Review by rubenzorrilla — APPROVED** [review]

**Pull request merged**